src/main/java/com/iota/iri/conf/SolidificationConfig.java

```diff
@@ -0,0 +1,22 @@
+package com.iota.iri.conf;
+
+/**
+ * 
+ * Configurations that should be used for the tip solidification process, 
+ * You can also completely disable the process.
+ */
+public interface SolidificationConfig extends Config {
+
+    /**
+     * @return {@value Descriptions#TIP_SOLIDIFIER_ENABLED}
```

Comment:
Can you also add default value like in:
https://github.com/iotaledger/iri/pull/1159 (a PR that will be merged after LS is released)